cmd/api/src/model/appcfg/flag.go

```diff
@@ -125,6 +126,13 @@ func AvailableFlags() FeatureFlagSet {
 			Enabled:       false,
 			UserUpdatable: true,
 		},
+		FeatureAutoTagT0ParentObjects: {
```

Comment:
This will need an explicit migration in a .sql file once this PR gets merged into main https://github.com/SpecterOps/BloodHound/pull/772

Reply:
gotcha, so that means I should set the "My changes include a database migration" flag?

Reply:
Correct. I believe we will be merging #772 for next release so this changeset should include a migration and have it be reflected in the PR description.